src/data/dem_data.js

```diff
@@ -44,7 +44,7 @@ register('Level', Level);
 // surrounding pixel values to compute the slope at that pixel, and we cannot accurately calculate the slope at pixels on a
 // tile's edge without backfilling from neighboring tiles.
 
-class DEMData {
+export default class DEMData {
```

Comment:
I think we should avoid mixing default and named exports https://github.com/mapbox/mapbox-gl-js/issues/6336